labcontrol/gui/js_tests/addPlateModal_tests.html

```diff
@@ -0,0 +1,34 @@
+<!--
+/**
+ * Loosely based on The Emperor Project's javascript testing framework
+ * by Yoshiki Vazquez Baeza
+ */
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+    <title>addPlateModal.js Unit Tests</title>
+    <link rel="stylesheet" href="qunit/qunit-2.9.2.css" type="text/css"
+          media="screen">
+
+    <!-- Testing framework requirements -->
+    <script src="qunit/qunit-2.9.2.js"></script>
+
+    <!-- LabControl scripts generally require JQuery -->
+    <script src="../static/vendor/js/jquery-3.2.1.min.js"></script>
+
+     <!-- Library under test requires DataTables -->
+    <script src="../static/vendor/js/datatables.min.js"></script>
+
+    <!-- Library under test -->
+    <script src="../static/js/addPlateModal.js"></script>
+
+    <script src="addPlateModal_tests.js"></script>
+</head>
+<body>
+<div id="qunit"></div>
+<div id="qunit-fixture">
+</div>
+</body>
+</html>
```

Comment:
according to GitHub, the last line in this file doesn't have a newline. Not concerned about it, personally.